src/services/resource.js

```diff
@@ -79,17 +79,36 @@ module.exports = class resourceHelper {
 			}
 
 			const uniqueOrganizationIds = [...new Set(resources.map((item) => item.organization_id))]
+			const uniqueCreatorIds = [...new Set(resources.map((item) => item.user_id))]
+
 			const orgDetailsResponse = await userRequests.listOrganization(uniqueOrganizationIds)
+			const userDetailsResponse = await userRequests.list(
+				common.FILTER_ALL.toLowerCase(),
+				1,
+				uniqueCreatorIds.length,
+				'',
+				organization_id,
+				{ user_ids: uniqueCreatorIds }
+			)
+
 			let orgDetails = {}
+			let userDetails = {}
 			if (orgDetailsResponse.success && orgDetailsResponse.data?.result?.length > 0) {
 				orgDetails = _.keyBy(orgDetailsResponse.data.result, 'id')
 			}
 
+			if (userDetailsResponse.success && userDetailsResponse.data.result.data.length > 0) {
```

Comment:
@adithyadinesh0412  make this triggger error if some if the result or data is not there plesae make this like orgDetails condition